.coveragerc

```diff
@@ -5,5 +5,7 @@ branch = True
 include_namespace_packages = true
 
 # might also be convenient to add one of these:
-omit = **/__init__.py
+omit =
+    **/__init__.py
+    sandbox
```

Comment:
sorry but where `sandbox` folder is use?

Reply:
I think there was some code that was unused and I moved that into sandbox folder (experimental code) in a previous PR.